examples/cmake/add_subdir/dummy.cpp

```diff
@@ -0,0 +1,32 @@
+#include <thrust/detail/config.h>
```

Comment:
Why is this a .cpp file?

Reply:
It's not part of thrust, it's part of a example project that demonstrates how to use the add_subdirectory(...) stuff to build an application that conditionally builds applications with different backends, one cpp and one cuda. I figured it be helpful to show the common trick of `#include <some_file.cpp>` from `some_file.cu` when doing that sort of thing.